resources/assets/coffee/_classes/tooltip-beatmap.coffee

```diff
@@ -0,0 +1,63 @@
+###
+#    Copyright 2015-2017 ppy Pty. Ltd.
+#
+#    This file is part of osu!web. osu!web is distributed with the hope of
+#    attracting more community contributions to the core ecosystem of osu!.
+#
+#    osu!web is free software: you can redistribute it and/or modify
+#    it under the terms of the Affero GNU General Public License version 3
+#    as published by the Free Software Foundation.
+#
+#    osu!web is distributed WITHOUT ANY WARRANTY; without even the implied
+#    warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#    See the GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with osu!web.  If not, see <http://www.gnu.org/licenses/>.
+###
+
+class @TooltipBeatmap
+  constructor: ->
+    $(document).on 'mouseover', '[data-beatmap-title]', @onMouseOver
```

Comment:
This is likely going to cause unintentional triggering on elements that end up with `data-beatmap-title` on them; use some `js-` prefixed selector like `js-tooltip-beatmap` instead